common/cultures/culture_vrykul.txt

```diff
@@ -10,24 +10,18 @@ vrykul_group = {
 
 		# Names
 		male_names = {
-			Aafrark Akonund Akulbent Athnenelf Berurdr Bjig Braagard Bridarbolf Brun Cargirs Ceskum Cugnonnont Deluskand Dokognelf Fidhuthurdr Firatald Fjafrak Fjildalid Aderigg Akalbelf Asgrul Bjaard Bjall
-			Bjalordgrom Bjatendod Bjill Bjodho Bjoktizim Bjukold Bjull Borlutholf Brakerr Brik Brithovird Brollarvel Brunvulf Cak Cur Deskong Deygnveg Dod Erles Ethnifik Faaktor Felf Fennargirk Fidald Fjegnherk
-			Fjokkogniw Fjonom Fjorognerd Fjorr Fjunnve Fjurbrirn Fonnafaw Fulfr Ghratarbam Ghrent Ghrutont Gjudrill Gjuk Gjyll Glew Glilbererd Glokiges Hargug Hellskihom Hjadrilf Hjer Hjill Hjitonnond Hjoemidilf
-			Hjurd Hoes Hurborholl Fuycall Ghruyrdr Gidelbarr Ging Gjarkomeld Gjarn Gjerdriw Gjilsta Gjim Glamaenoll Glan Glunnuho Gorjaw Gorlord Gulbird Gulrang Guram Hifacelfr Hint Hjannond Hjong Hjukurdgris Hjurjaniw
-			Hjuyrs Hont Hovoesviw Hrarnemarn Hrefarn Hreldgutord Hrund Hrurjuva Hruyrs Ildgu Imdorn Irbrarhurn Iredrag Kar Kefoeldid Keseld Kork Milment Reng Roldgek Sithidril Skamanin Skel Skirn Skothihirn Slelskar Slithinurn Sordiw
-			Slod Slullskinnull Slundell Jannuyrdrom Jannvekke Jig Jillskog Jyrr Kanilfr Kerlerdr Kirn Koemderr Kog Kork Kulro Migg Minog Mirkamak Mognvuthom Murk Olskannerdr Othemill Rasgrar Rennlag Reylorhegg Rilren Rordun Rud Rulmifi
-			Skaad Skaktuk Skeri Skim Skoethang Skong Skor Skurvoll Slannvor Sleind Slilbreld Slinvent Slugegnha Slund Slunnvurs Slurdruvogg Stand Stil Stirholgog Stulf Sugnvofo Sval Svelburn Tek Thokuw Thuktern Thundellelfr Tulbrurhund
-			Tundunnus Ukegg Vinnirdr Votathelfr Vurve Yant Yerdr Yilf Yow Slusek Stardurn Stogazand Storhumdold Stud Stukto Stylstagoll Svelsto Sving Tant Teilfr Tell Ters Therlared Thethnos Thiljing Thillondin Ugalgew Ulbung Uthodhal Uzenvaw
-			Vaaceno Vodreg Vugnhumoll Vuzadhull Yord Yow Yurd
+			Alf Algar Andjor Angantyr Angvold Arni Bagge Balagarde Banahogg Bjorn Brandolf Brinvulf Brulf Brvet Brytag Calder Dagr Drengr Durnolf Dyre Egil Erling
```

Comment:
Ooof, there are fewer than before.
Me not like.

Reply:
There is less of them in general but they are all fitting while the old ones are almost all wrong

Reply:
> There is less of them in general but they are all fitting while the old ones are almost all wrong

I don't get how it affects the fact there are fewer names.

Reply:
Whats wrong with having fewer names?

Reply:
> Whats wrong with having fewer names?

What is wrong with having as you said wrong names?

Reply:
Everyone including me wants more names because it increases diversity.

Reply:
So it's ok if i copy human names and put them in vrykul namelist?